src/sonic-host-services-data/templates/common-auth-sonic.j2

```diff
@@ -23,6 +23,7 @@ auth	[success=1 default=ignore]	pam_tacplus.so server={{ last_server.ip }}:{{ la
 
 {% endif %}
 {% elif auth['login'] == 'tacacs+' or auth['login'] == 'tacacs+,local' %}
+auth    [default=ignore system_err={{servers | count}}] pam_exec.so debug stdout type=auth /usr/local/bin/check_system
```

Comment:
better to introduce a new mode, not enable this by default for public.

Reply:
done